doc/ref_impl/cntt-ri2/chapters/chapter01.md

```diff
@@ -1,7 +1,6 @@
-[<< Back](../)
+﻿# 1. Overview
 
-# 1. Overview
-<p align="right"><img src="../figures/bogo_dfp.png" alt="scope" title="Scope" width="35%"/></p>
+![State](../figures/bogo_dfp.png)
```

Comment:
```suggestion
![State](../figures/bogo_dfp.png) <!-- width="35" -->
```
Let's use this notation for now, so we have a chance to scale the images during the rst conversion.